e2e/interchaintestv8/chainconfig/chain_config.go

```diff
@@ -14,8 +14,8 @@ var DefaultChainSpecs = []*interchaintest.ChainSpec{
 			ChainID: "simd-1",
 			Images: []ibc.DockerImage{
 				{
-					Repository: "ghcr.io/cosmos/ibc-go-wasm-simd", // FOR LOCAL IMAGE USE: Docker Image Name
-					Version:    "feat-ibc-eureka",                 // FOR LOCAL IMAGE USE: Docker Image Tag
+					Repository: "ghcr.io/cosmos/ibc-go-wasm-simd",  // FOR LOCAL IMAGE USE: Docker Image Name
+					Version:    "gjermund-7591-ics20-abi-encoding", // FOR LOCAL IMAGE USE: Docker Image Tag
```

Comment:
This one can only be updated once https://github.com/cosmos/ibc-go/pull/7592 is merged. We could wait, but I'd rather just update this in a PR so we're not blocked on ibc-go reviewing which might take time.

Reply:
Hmm, should it be merged soon ahead of the security audit?

Reply:
Yes, I would think so.